### PR TITLE
Converter logic

### PR DESCRIPTION
**Pull request by OlaniyiOshinusi**

### CODE REVIEW

File: src/helper.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const convertToNum = (val) => parseInt(val);
```
Comment (jcsaenzcarrasco) [inline src/helper.js:1]: I don't think you now have the function convertToNum, you should not test it

File: src/App.test.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ test("the initial input box is empty", ()=>{
   render(<App/>);
   const celsiusInput = screen.getByRole('textbox');//Gets the input box and assigns it to a variable
```
Comment (jcsaenzcarrasco) [inline src/App.test.js:6]:
```suggestion

```

**Review by jcsaenzcarrasco — CHANGES_REQUESTED** [review]
You need to delete code that is no longer functional

**Review by alexcasper — APPROVED** [review]
Seems ok to me!

**Review by missKatiaPunter — APPROVED** [review]

**Review by jcsaenzcarrasco — APPROVED** [review]

**Pull request merged**